Готовый проект после ревью.

3elt3er opened the pull request.

3elt3er committed:
Готовый проект после ревью.

maximzadorkin commented:
До сих пор можно мержить без апрува. Надо поправить

maximzadorkin commented on index.html:
```diff
@@ -2,28 +2,39 @@
         <title>Adilgereev CV</title>
         <link rel="icon" href="favicon.ico" type="image/x-icon">
         <link rel="stylesheet" href="styles.css"/>
+        <link rel="stylesheet" href="stylesA.css"/>
```
Так не называем стили. Как я должен понять, придя в проект через полгода, что лежит в этом файле? Какие стили? Что за загадочная буква А?

maximzadorkin commented on index.html:
```diff
@@ -2,28 +2,39 @@
             
                    
-                    <img src="Me.jpg" alt="Здесь должна быть фотка меня" class="PhotoMe"/>
+                    <img src="Me.jpg" alt="Ошибка отоброжении фотографии пользователя" class="PhotoMe"/>
```
Можно и менее официально в целом) самое главное просто осмысленно. Но это на будущее тебе, исправлять не надо)

maximzadorkin commented on index.html:
```diff
@@ -2,28 +2,39 @@
                   </ul>
-                    <p>Москва, не готов к переезду, готов к командировкам за границу</p>
-                    <p>Указан примерный район поиска работы:</p>
+                    <p id="discription">Москва, не готов к переезду, готов к командировкам за границу</p>
```
Id - идентификатор. Идентификатор — это уникальная вещь. А она у тебя присвоена разным тегам одна и та же

3elt3er replied:
а зачем тогда индетификаторы? типо можно же просто в файле htlml написать стайл.

maximzadorkin commented on map.html:
```diff
@@ -5,6 +5,6 @@
     </head>
     <body>
-       <img src="Moscow Map.png" width="800" hspace="350" alt="Здесь должна быть фотка карты"> 
+       <img src="Moscow Map.png" width="800" hspace="350" alt="Ошибка отоброжении фотографии карты поиска работы"> 
```
За русским языком следим) "отображения"

3elt3er replied:
XD, я заметил кстати это после пр, но исправлять и заливать новый пр не хотелось))

maximzadorkin commented on styles.css:
```diff
@@ -23,13 +27,14 @@ body {
 }
-.intro p{
+
+#discription{
```
Стили по идентификатору лучше не задавать, учти, что это просто для тренировки и на всякий случай знать, что так можно делать при необходимости. Поправлять не нужно

maximzadorkin commented on styles.css:
```diff
@@ -154,9 +142,21 @@ a:visited {
 
 .button:hover {
     opacity: 0.9;
+    border: 1px solid rgb(113, 32, 108);
```
На будущее лучше в едином стиле. Поправлять не нужно, для изучения хорошо, но лучше какую схему выбрал той и придерживайся. Для красоты и понятности кода

maximzadorkin replied:
Так все же rgba или hex? а?

maximzadorkin commented on styles.css:
```diff
@@ -154,9 +142,21 @@ a:visited {
+    outline: none;
+}
+
+.button:focus {
```
Не могу проверить без компа. Что происходит при нажатии на так на странице?

3elt3er replied:
Теперь всё окей. Но, бордер сайзинг не помог, почему то всё равно рамка не считалась как новая. Я добавил просто в родительской кнопке бордер 1 пиксель, соответственно когда происходит какое то действие, кнопка не увеличивается, а просто добавляет этот бордер 1 пиксель, но так как ранее уже был задан размер на +1, изменений не видно. Согласен, костыль, но я хз как по-другому.

maximzadorkin commented on styles.css:
```diff
@@ -1,10 +1,14 @@
 
```
Не вижу использования грид

3elt3er replied:
Грид в следующем пр((((9

maximzadorkin reviewed (changes requested):
Дорабатываем и новый коммит в этом пр

maximzadorkin commented:
Ещё почитай о позиционировании в css. Добавь использование relative, absolute, static, sticky.

3elt3er committed:
New changes

1. Positioning
2. Navigation (in progress)
3. Remarks from review.

3elt3er committed:
Новые изменения

Оптимизация, grid, позиционирование, якорные ссылки - навигация. Небольшие изменения элементов.
Проблема со sticky, так как бокс навигации перекрывает пол экрана.

3elt3er committed:
Небольшая оптимизация для маленьких телефонов

Переместил выделение для маленьких телефонов, вообще я бы не так текст выделил, если бы не делал фикс, я бы просто создал отдельный див с блоком с текстом 100 000 руб., и также с абсолютом и фиксом его бы вставил на то место. Не знаю, поменялось бы то, что на маленьких телефонах всё скачет, но мб так было бы лучше, потому что при уменьшении экрана текст скачет то вверх то вниз, то половина суммы на следующую строчку переносится.

maximzadorkin commented:
Вынеси в assets png

maximzadorkin commented:
Arrow должна все же быть в svg формате

maximzadorkin commented on map.html:
```diff
@@ -5,6 +5,6 @@
     </head>
     <body>
-       <img src="Moscow Map.png" width="800" hspace="350" alt="Здесь должна быть фотка карты"> 
+       <img src="Moscow Map.png" width="800" hspace="350" alt="Ошибка отображения фотографии карты поиска работы"> 
```
Строка получается очень длинная. Сделай примерно так:  
<img  
        src=""  
        width=""  
        ...  
/>

maximzadorkin replied:
Посмотри как это выглядит на маленьком экране. Появляется скролл и пустые места справа и слева. Не очень хорошо

maximzadorkin commented on index.html:
```diff
@@ -2,53 +2,75 @@
+
+
+
+    <div class="card bg-blue" id="1">
```
Про id я вроде писал, что не следует так бездумно называть

maximzadorkin commented on index.html:
```diff
@@ -157,6 +182,13 @@ <h2>Связаться со мной</h2>
     
      </form>
     </div>
+    <div class="positioning">
```
Так класс точно называть не следует. Переработай названия, чтобы они отражали не css свойства, а семантическое назначение

3elt3er replied:
Тяжеловато пока названия придумывать, если честно(

maximzadorkin commented on index.html:
```diff
@@ -157,6 +182,13 @@ <h2>Связаться со мной</h2>
+    <div class="positioning">
+      <div class="button_fixed">
+        <a href="#PhotoMe">
+         <img src="arrow_up.png" alt="Ошибка отображения стрелки навигации" id="arrow"/> 
```
Про длинну строки уже писал. Везде посмотри. Обычно ограничиваемся 120 символами. Но все же если есть возможность отделяй поаккуратнее, чтобы символов в 80 умещаться, каждый атрибут можно с новой строки написать в данном случае, так как читать неудобно

maximzadorkin commented on styles.css:
```diff
@@ -1,35 +1,41 @@
 
-@import url('https://fonts.googleapis.com/css2?family=Poppins&family=Roboto+Slab&display=swap');
-
+@font-face {
```
Файл .ttf вынеси в fonts папку или как-то так. Не нужно все в одном месте складировать

maximzadorkin commented on styles.css:
```diff
@@ -1,35 +1,41 @@
     
     background-image: url(dust_scratches.webp);
     font: normal 16px 'Roboto Slab', serif;
+    
```
Касаемо таких отступов писал уже тебе в другом пр. Тут все то же доработай. Стиль должен быть единым, а у тебя постоянные отступы хаотичные. Выглядит нехорошо, сбивает с толку при чтении

maximzadorkin commented on styles.css:
```diff
@@ -1,35 +1,41 @@
     .intro {
         text-align: center;
         display: block;
+        padding: 20px;
```
media запросы стоит выносить в конец файла, чтобы не путаться

maximzadorkin commented on styles.css:
```diff
@@ -1,35 +1,41 @@
 }
-.intro p{
+
+.discription{
```
Кажется надо немного подтянуть английский

maximzadorkin commented on styles.css:
```diff
@@ -39,51 +45,41 @@ body {
     padding: 15px 20px;
     margin-bottom: 20px;
     border-radius: 20px;
+    box-shadow: 4px 4px 14px rgba(0,0,0,0.1);
```
Помним о style code. Я бы написал так rgba(0, 0, 0, 0.1). С пробелами между ,

maximzadorkin commented on styles.css:
```diff
@@ -151,12 +147,131 @@ a:visited {
+
+
+.positioning{
+position: relative;
```
Тут ты даже про табуляцию забыл

maximzadorkin commented on styles.css:
```diff
@@ -151,12 +147,131 @@ a:visited {
+}
+
+
+@media (max-width:784px){
```
Вот сейчас вдвойне странно выглядит, что часть медиа запросов у тебя сверху, а основная часть здесь. Все в одну кучу рассортируй

maximzadorkin commented on styles.css:
```diff
@@ -151,12 +147,131 @@ a:visited {
+    height: 18%;
+    border-radius: 20px;
+    background-color: #ff000040;
+    right: 50px;
```
В другом PR тебе писал, что лучше все-таки стили в определенном порядке писать. Сначала все, что к контейнеру относится, потом его красочное оформление, потом оформление текста, потом состояния и тд

maximzadorkin reviewed (changes requested):
Проблемы в основном те же, что и в layoute. Следи за стилем кода

maximzadorkin reviewed (changes requested):
Маленькое замечание по визуалу

3elt3er committed:
Добавлена стилистика кода + небольшие исправления недостатков

3elt3er committed:
Переписан CSS на SASS, исправлена стилистика кода

3elt3er committed:
Изменено позиционирование блоков с зарплатой, изменены директории, стилистика кода

maximzadorkin commented:
А я все еще могу мержить без апрува, ай яй яй

maximzadorkin commented on index.html:
```diff
@@ -2,28 +2,39 @@
                 <div class="info-text">
                     <h1>Адилгереев Артем Максимович</h1>
-                    <p>Мужчина, 19 лет, родился 15 октября 2002</p>
+                    <p id="discription">Мужчина, 19 лет, родился 15 октября 2002</p>
```
Мои глаза наполнены скорбью и печалью
Все же dEscription(

maximzadorkin replied:
И кстати ты так везде вальнул

maximzadorkin commented on styles.css:
```diff
@@ -103,7 +91,7 @@ a:visited {
+    color: #723333;
 }
 
 .contact h2{
```
Хоба
Помнишь про стиль кода писал? У тебя почему то половина кода с h2{, а вторая половина с h2 {

maximzadorkin commented:
Давай исправляй недочеты


И следующий пр должен быть с sass, а так же уберем стили по атрибутам, по тегам в тегах и прочее, что ты делал исключительно для обучения

maximzadorkin commented:
Ах да, в следующем ПР настрой github pages

3elt3er committed:
Дописан SASS. Исправление багов.

3elt3er committed:
Убрал программу сборщик Prepros, переписал с использованием Gulp

maximzadorkin commented on build/styles/styles.css:
```diff
@@ -0,0 +1,197 @@
+@font-face {
```
build папка осталась, убери

maximzadorkin commented on index.html:
```diff
@@ -1,162 +1,213 @@
+      <div class="card bg-green" id="main info">
+        <h2>Обо мне</h2>
+        <span>Мой профиль на github:</span>
+        <a href="https://github.com/3elt3er/" target="_blank">Нажми</a></li>
```
что за </li> в конце?

maximzadorkin commented on package.json:
```diff
@@ -0,0 +1,29 @@
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
мне кажется это надо убрать

maximzadorkin commented on src/sass/styles.sass:
```diff
@@ -0,0 +1,169 @@
+    h2
+        margin-bottom: 10px
+
+.bg-blue 
```
с  bg вложенность сделай как сделал с card

maximzadorkin commented on src/sass/styles.sass:
```diff
@@ -0,0 +1,169 @@
+    padding: 15px 20px
+    border: 1px solid transparent
+    border-radius: 20px
+    margin-top: 10px
```
css порядок нарушен

maximzadorkin commented on src/sass/styles.sass:
```diff
@@ -0,0 +1,169 @@
+    margin: auto
+    padding-top: 30px
+    padding-bottom: 30px
+    gap: 20px
```
это относится к флексам
